README.md

```diff
@@ -38,41 +43,102 @@ on to create a full desktop environment.
 Labwc tries to stay in keeping with [wlroots] and [sway] in terms of general
 approach and coding style.
 
-Labwc only understands [wayland-protocols] &amp; [wlr-protocols], and it cannot
-be controlled with dbus, sway/i3-IPC or other technology. The reason for this is
-that we believe that custom IPCs and protocols create a fragmentation that
-hinders general Wayland adoption.
+Labwc has no reliance on any particular Desktop Environment, Desktop Shell or
+session. Nor does it depend on any UI toolkits such as Qt or GTK.
+
+### 1.1 Why?
+
+Firstly, we believe that there is a need for a simple Wayland window-stacking
+compositor which strikes a balance between minimalism and bloat approximately
+at the level where Window Managers like Openbox reside in the X11 domain.
+This belief has been borne out of the X11 computing background of most of the
+core-devs which is one of low resource Desktop Environments such Mate/XFCE or
+standalone Window Managers like Openbox with no reliance on heavy eco-systems.
+
+Secondly, the Wayland community has achieved an amazing amount so far, and we
+want help to solve the unsolved problems to make Wayland viable for more
```

Comment:
Was this supposed to be "we want to help solve"?

"We want help to solve" could also be correct but changes the meaning a bit (we are looking for someone to help us, rather than trying ourselves to offer help).

Reply:
Thanks good spot. Should be "to help" (we want to help...)